extensions/iam/decentralized-identity/identity-did-service/src/main/java/org/eclipse/dataspaceconnector/identity/DecentralizedIdentityService.java

```diff
@@ -59,7 +58,7 @@ public DecentralizedIdentityService(DidResolverRegistry resolverRegistry, Creden
 
     @Override
     public Result<TokenRepresentation> obtainClientCredentials(TokenParameters parameters) {
-        var jwt = VerifiableCredentialFactory.create(privateKey, issuer, parameters.getAudience(), clock);
+        var jwt = JwtUtils.create(privateKey, issuer, issuer, parameters.getAudience(), clock);
```

Comment:
micro nit: static import?

Reply:
Are you sure? It seems the version without static import is clearer

Reply:
for me, reading `var jwt = create(...)` leaves little room for interpretation :)